src/main/java/frc/robot/subsystems/pivot/PivotIOTalonFX.java

```diff
@@ -231,7 +246,12 @@ public void setPivotFromSpeakerDistance(double speakerDistance) {
     setPivotAngle(speakerAngle);
   }
 
-  /** Sets Pivot based on Pass Distance */
+  /**
+   * Uses distance in meters from the passing position to set the pivot angle (degrees) of the
+   * shooter
+   *
+   * @param passDistance the distance in meters from the passing position
+   */
   @Override
   public void setPivotFromPassDistance(double passDistance) {
     double passAngle = passAngleLookupValues.getLookupValue(passDistance);
```

Comment:
heres and idea, which i believe i mentioned before. Why don’t you move the setPivotFromDistance() stuff into pivot.java. then both the sim and real will have exactly the same functionality.